pynobo.py

```diff
@@ -849,6 +849,31 @@ async def async_update_zone(self, zone_id, name=None, week_profile_id=None, temp
 
         await self.async_send_command(command)
 
+    def update_week_profile(self, week_profile_id, name=None, profile=None):
+        self._create_task(self.async_update_week_profile(week_profile_id, name, profile))
+
+    async def async_update_week_profile(self, week_profile_id, name=None, profile=None):
```

Comment:
For completeness, could you add add and delete as well?

Reply:
I see we don't have add and delete for zones, so I wouldn't say this is a blocker.

Reply:
so there is now a add routine